doc/nrf/libraries/bluetooth_services/mesh/light_ctrl_cli.rst

```diff
@@ -21,6 +21,151 @@ Persistent storage
 
 None.
 
+Shell commands
+**************
+
+The Bluetooth mesh shell subsystem provides set of commands to interract with the Light LC Client model instantiated on a device.
+
+To make these commands available, enable :kconfig:`CONFIG_BT_MESH_SHELL` and :kconfig:`CONFIG_BT_MESH_SHELL_LIGHT_CTRL_CLI`.
+
+``mdl_ctrl instance get-all``
+-----------------------------
+
+	Print all instances of the Light LC Client model on the device.
+
+
+``mdl_ctrl instance set <elem_idx>``
+------------------------------------
+
+	Select the Light LC Client model instantiated on the specified element id. This instance will be used in message sending. If no model instance was selected, the first model instance found on the device will be used by default.
+
+	* ``elem_idx``: Element index where to find the model instance.
+
+
+``mdl_ctrl mode-get``
+---------------------
+
+	Get a Light Lightness Control Server's current Mode.
+
+
+``mdl_ctrl mode-set <enable | disable>``
+----------------------------------------
+
+	Set a Light Lightness Control Server's current Mode and wait for a response.
+
+	* ``enable | disable``: Mode to set.
+
+
+``mdl_ctrl mode-set-unack <enable | disable>``
+----------------------------------------------
+
+	Set a Light Lightness Control Server's current Mode without requesting a response.
+
+	* ``enable | disable``: Mode to set.
+
+
+``mdl_ctrl occupancy-get``
+--------------------------
+
+	Get a Light Lightness Control Server's current Occupancy Mode.
+
+
+``mdl_ctrl occupancy-set <enable | disable>``
+----------------------------------------------
+
+	Set a Light Lightness Control Server's current Occupancy Mode and wait for a response.
+
+	* ``enable | disable``: Occupancy Mode to set.
+
+
+``mdl_ctrl occupancy-set-unack <enable | disable>``
+----------------------------------------------------
+
+	Set a Light Lightness Control Server's current Occupancy Mode without requesting a response.
+
+	* ``enable | disable``: Occupancy Mode to set.
+
+
+``mdl_ctrl light-onoff-get``
+----------------------------
+
+	Get a Light Lightness Control Server's current OnOff state.
+
+
+``mdl_ctrl light-onoff-set <onoff> [transition_time_ms  [delay_ms]]``
```

Comment:
```suggestion
``mdl_ctrl light-onoff-set <onoff> [transition_time_ms [delay_ms]]``
```
Nitpick:
I see that there is differences in the use of space for the parameters. E.g. the `mdl_loc local-set` uses single space between params. This applies for several files.